File: java/quarkus/sentry/src/main/java/org/acme/GreetingResource.java
package org.acme;

import org.slf4j.Logger;
import org.slf4j.LoggerFactory;

import javax.ws.rs.GET;
import javax.ws.rs.Path;
import javax.ws.rs.Produces;

import static javax.ws.rs.core.MediaType.TEXT_PLAIN;

@Path("/hello-resteasy")
public class GreetingResource {

    private static final Logger LOGGER = LoggerFactory.getLogger(GreetingResource.class);

    @GET
    @Produces(TEXT_PLAIN)
    public String hello() {
        LOGGER.info("Hello with Info!");
        LOGGER.error("Hello with Error!");
        LOGGER.warn("Hello with Warning!");
        return "Hello RESTEasy";
    }
}

File: java/quarkus/getting-started-kotlin/src/main/kotlin/org/acme/GreetingResource.kt
package org.acme

import org.eclipse.microprofile.config.inject.ConfigProperty
import javax.inject.Inject
import javax.ws.rs.GET
import javax.ws.rs.Path
import javax.ws.rs.Produces
import javax.ws.rs.core.MediaType.TEXT_PLAIN

@Path("/hello")
class GreetingResource {

    @Inject
    lateinit var greetingConfig: GreetingConfig

    @ConfigProperty(name = "greeting.name", defaultValue = "abc")
    lateinit var name: String

    @ConfigProperty(name = "greeting.base64Name", defaultValue = "abc")
    lateinit var base64Name: Base64Value

    @GET
    @Produces(TEXT_PLAIN)
    fun hello(): String = "hello $name"

    @GET
    @Path("/base64")
    @Produces(TEXT_PLAIN)
    fun helloBase64() = "hello $base64Name"

    @GET
    @Path("/config")
    @Produces(TEXT_PLAIN)
    fun helloGreetingConfig() = "hello ${greetingConfig.prefix} ${greetingConfig.suffix}"
}
